packages/modules/cra-template-desktop-viewer/template.json

```diff
@@ -0,0 +1,78 @@
+{
+  "package": {
+    "dependencies": {
```

Comment:
I think we can have devDeps now right? Or is that not until 4.0?

Reply:
yes....just wasn't sure if we want to go there yet since people using older versions would not have a working template....but I guess 4.x has been around a while now and we promote npx in all of our docs so I will make the change

Reply:
well looks like we don't support 4.x yet so this will have to wait